### DISCUSSION TITLE
Need help for swissup navigation pro module

### DISCUSSION THREAD

**Discussion by JamesDenish** [breezefront/community#261] [1. Help] [answered]
Hello @vovayatsyuk 

One of our client is looking to have slider as a menu which we have tried and got succeed well using NavigationPro extension used in below site I have attached a lik here :- [](https://www.coplax-verpackungen.ch/de/)

Issue:
Menu is not loading according to requirements we have made changes along with loading timeout and where ever needed but still it sometimes disappear as below screenshot in chrome and safari so kindly I request you to help us.

Thanks in advance.
![IMG_1068](https://github.com/breezefront/community/assets/142228975/eef31370-38ff-48d8-9300-65840b1ecd76)
![Snag_1500f538](https://github.com/breezefront/community/assets/142228975/8f2f1b31-2c85-440a-8561-d2f7ec87dd5b)

Thank you
JamesDenish

**Comment by vovayatsyuk** [comment, marked as answer]
You need to remove `async` attribute from your script:

```html
<script type="text/javascript" async="async" src=".../Vendor/theme/de_CH/js/extend.js"></script>
```

Replace it with `defer` instead.

p.s. How did you add your js script? We have a guide on how to do that: https://breezefront.com/docs/custom-javascript

**Comment by JamesDenish** [comment]
Hello @vovayatsyuk 

We have added script below way in child-theme in Magento_Theme/layout/default_head_blocks.xml 
```
<?xml version="1.0"?>
<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
    <head>
         <script async="async" src="js/extend.js"  />
         <css rel="preload" as="style"  src="css/extend.css"/>
    </head>
    <body>
        <referenceBlock name="breeze.js">
            <arguments>
                <argument name="bundles" xsi:type="array">
                    <item name="cms" xsi:type="array">
                        <item name="active" xsi:type="boolean">true</item>
                    </item>
                </argument>
            </arguments>
        </referenceBlock>
    </body>
</page>
```
We have changed async to defer as well so kindly advice on the above snippet.
Below is the updated snippet after replacing defer
`<?xml version="1.0"?>
<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
    <head>
         <script defer="defer" src="js/extend.js"  />
         <css rel="preload" as="style"  src="css/extend.css"/>
    </head>
    <body>
        <referenceBlock name="breeze.js">
            <arguments>
                <argument name="bundles" xsi:type="array">
                    <item name="cms" xsi:type="array">
                        <item name="active" xsi:type="boolean">true</item>
                    </item>
                </argument>
            </arguments>
        </referenceBlock>
    </body>
</page>
`
Thanks in advance...
Thank You
JamesDenish
